src/modules/settings/components/settings/youtube/EmoteMenu.jsx

```diff
@@ -28,5 +32,5 @@ registerComponent(EmoteMenu, {
   settingId: SettingIds.EMOTE_MENU,
   name: SETTING_NAME,
   category: CategoryTypes.CHAT,
-  keywords: ['twitch', 'emotes', 'popup'],
+  keywords: ['youtube', 'emotes', 'popup'],
```

Comment:
did we duplicate just for a keyword? why not just remove twitch/youtube?

Reply:
The component differs, twitch has 3 configuration however youtube has 2.